init.zsh

```diff
@@ -2,14 +2,14 @@
 # Zim initializition
 #
 
-autoload -Uz is-at-least
-if ! is-at-least 5.2; then
-  print "ERROR: Zim didn't start. You're using zsh version ${ZSH_VERSION}, and versions < 5.2 are not supported. Update your zsh." >&2
-  return 1
-fi
+# autoload -Uz is-at-least
+# if ! is-at-least 5.2; then
+#   print "ERROR: Zim didn't start. You're using zsh version ${ZSH_VERSION}, and versions < 5.2 are not supported. Update your zsh." >&2
+#   return 1
+# fi
 
 # Define zim location
-ZIM="${ZDOTDIR:-${HOME}}/.zim"
+(( ! ${+ZIM_HOME} )) && ZIM_HOME="${ZDOTDIR:-${HOME}}/.zim"
```

Comment:
The spec is available here: https://www.freedesktop.org/wiki/Software/xdg-user-dirs/

So yeah, `XDG_CONFIG_HOME` is actually the correct way to check. It would become something like this:

```zsh
(( ! ${+ZIM_HOME} )) && \
  if [[ -s ${XDG_CONFIG_HOME:-"${HOME}/.config"}/zim/zimrc ]]; then
    ZIM_HOME=${XDG_CONFIG_HOME:-"${HOME/.config}"}/zim
  else
    ZIM_HOME="${ZDOTDIR:-${HOME}}/.zim"
  fi
```